### ISSUE TITLE
Low contrast in UI elements across event details

### ISSUE THREAD

**Issue by GauriLekshmiSathya** [fossunited/fossunited#1707]
# Description

Using WAVE (Website Accessibility Evaluation Tool), 77 contrast errors were flagged on the page: https://fossunited.org/dashboard/schedule/indiafoss/2025. 

<br>

<img width="2879" height="1544" alt="Image" src="https://github.com/user-attachments/assets/f177ff5c-1c38-495d-823f-c2b659c61c28" />

<br><br>

These errors are related to insufficient color contrast between foreground elements, such as text, icons, and other interface elements, and their respective backgrounds.

# Screenshots

A couple of examples are provided below for reference:

<br>

<img width="2879" height="1523" alt="Image" src="https://github.com/user-attachments/assets/261352b2-ee04-45c7-8fe8-d572b7a875b7" />

<br><br>

<img width="2877" height="1526" alt="Image" src="https://github.com/user-attachments/assets/569cc1d7-3ec9-49db-b9f5-fcf4885e4303" />

<br><br>

- Here the contrast issues are clearly demonstrated on the right side with the benchmarks it is measured against ("WCAG AA", "WCAG AAA", "Contrast Ratio").

**Comment by idlip** [comment]
hey gauri, thanks for bringing this up.

Most of the `/dashboard` pages use frappe-ui's preset color themes as it is (to reduce burden on us)
https://github.com/fossunited/fossunited/blob/develop/dashboard/node_modules/frappe-ui/src/tailwind/colors.json

Even our designer, jeswin had raised to use his own color palette to stay uniform.

I think we should look into overriding the color palette as per our theme and also, be compliant to WCAG.

Also we lag behind frappe-ui version #1437 
so we hope updating to latest would solve most of the issue, but instead we need our own flavor of colors.

Will try to take this as separate task or make it happen along with #1555 in case we cannot update the frappe-ui package.

Thank you!
